src/nvim/api/vim.c

```diff
@@ -2431,7 +2431,8 @@ Dictionary nvim_eval_statusline(String str, Dict(eval_statusline) *opts, Error *
 ///                 boolean attributes (such as |:command-bang| or |:command-bar|) set the value to
 ///                 "true". In addition to the string options listed in |:command-complete|, the
 ///                 "complete" key also accepts a Lua function which works like the "customlist"
-///                 completion mode |:command-completion-customlist|. Additional parameters:
+///                 completion mode |:command-completion-customlist|.
+///                 Additional parameters:
```

Comment:
We definitely want to at least link the docs somewhere here.

Reply:
done